input.json

```diff
@@ -166,237 +166,237 @@
+          "file": "./rawdata/Experiment-9/Trial-3_121231_yc1.csv",
           "relative_density": 0.47
         },
         {
           "id": 4,
-          "file": "Trial-4_130110_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-4_130110_yc1.csv",
           "relative_density": 0.40
         },
         {
           "id": 5,
-          "file": "Trial-5_130113_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-5_130113_yc1.csv",
           "relative_density": 0.39
         },
         {
           "id": 6,
-          "file": "Trial-6_130115_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-6_130115_yc1.csv",
           "relative_density": 0.43
         },
         {
           "id": 7,
-          "file": "Trial-7_130116_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-7_130116_yc1.csv",
           "relative_density": 0.41
         },
         {
           "id": 8,
-          "file": "Trial-8_130117_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-8_130117_yc1.csv",
           "relative_density": 0.49
         },
         {
           "id": 9,
-          "file": "Trial-9_130118_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-9_130118_yc1.csv",
           "relative_density": 0.41
         },
         {
           "id": 10,
-          "file": "Trial-10_130119_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-10_130119_yc1.csv",
           "relative_density": 0.37
         },
         {
           "id": 11,
-          "file": "Trial-11_130130_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-11_130130_yc1.csv",
           "relative_density": 0.44
         },
         {
           "id": 12,
-          "file": "Trial-12_130209_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-12_130209_yc1.csv",
           "relative_density": 0.47
         },
         {
           "id": 13,
-          "file": "Trial-13_130213_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-13_130213_yc1.csv",
           "relative_density": 0.48
         },
         {
           "id": 14,
-          "file": "Trial-14_130214_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-14_130214_yc1.csv",
           "relative_density": 0.44
         },
         {
           "id": 15,
-          "file": "Trial-15_130215_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-15_130215_yc1.csv",
           "relative_density": 0.50
         },
         {
           "id": 16,
-          "file": "Trial-16_130215-02_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-16_130215-02_yc1.csv",
           "relative_density": 0.47
         },
         {
           "id": 17,
-          "file": "Trial-17_130216_yc1.csv",
+          "file": "./rawdata/Experiment-9/Trial-17_130216_yc1.csv",
           "relative_density": 0.51
         },
         {
           "id": 18,
-          "file": "Trial-18_130222.csv",
+          "file": "./rawdata/Experiment-9/Trial-18_130222.csv",

```

Comment:
I don't think you need `./` in front of the rawdata. You can just do `rawdata/Experiment...`